static/js/components/tiles/line_tile.tsx

```diff
@@ -116,17 +124,17 @@ export const fetchData = async (props: LineTilePropType) => {
   return rawToChart(resp.data, props);
 };
 
-function draw(
+export function draw(
   props: LineTilePropType,
   chartData: LineChartData,
-  svgContainer: React.RefObject<HTMLElement>
+  svgContainer: HTMLElement
 ): void {
   const elem = document.getElementById(props.id);
   // TODO: Remove all cases of setting innerHTML directly.
   elem.innerHTML = "";
   const isCompleteLine = drawLineChart(
     props.id,
-    elem.offsetWidth,
+    elem.offsetWidth || props.svgChartWidth,
```

Comment:
hm, did you want props.svgChartWidth to take priority or elem.offsetWidth? I would guess props.svgChartWidth because if you pass a width, maybe would expect it to be respected?

Reply:
Good point. I am using it as a supplement since offsetWidth is 0 in jsdom. But good to swap.

Also I think for all the svg width, height, they should be explicitly passed in the draw function. Now it's a mix.